`clientSecret` is not required if token is for "Chrome App" (#46)
fregante/chrome-webstore-upload: How to generate Google API keys.md

[chrome-webstore-upload](https://github.com/DrewML/chrome-webstore-upload) uses the Chrome Web Store API. 

Here's how to get its 2 access keys: `clientId`, `refreshToken`

*Note:* the names you enter here don't really matter. This will take approximately 10 minutes, sorry.

1. Visit https://console.developers.google.com/apis/credentials

> 2. Create a project:

Reviewer: They're all 0, no need to update them
```suggestion
0. Create a project:
```

0. Visit https://console.developers.google.com/apis/credentials
0. Click **Create credentials** > **OAuth client ID**:

	> <img width="771" alt="Create credentials" src="https://user-images.githubusercontent.com/1402241/77865679-e89f3a00-722f-11ea-942d-5245091f22b8.png">

0. Select **Chrome App**, enter `Chrome Webstore Upload`, your extension’s ID, and click **Create** 

	> <img width="547" alt="Create OAuth client ID" src="https://user-images.githubusercontent.com/1402241/106205904-de6a0700-6184-11eb-8591-984e69c5e82a.png">

0. Save your ✅ `clientId` and ignore the `clientSecret`; `clientId` is 1 of 2 keys you will need.
0. Place your `clientId` in this URL and open it:

> 	`https://accounts.google.com/o/oauth2/auth?client_id=YOUR_CLIENT_ID&response_type=code&scope=https://www.googleapis.com/auth/chromewebstore&redirect_uri=urn:ietf:wg:oauth:2.0:oob&access_type=offline&approval_prompt=force`

Reviewer: > The API in this case _does not_ return `refresh_token`, but just the `access_token`.

Maybe this was my fault, I did not use this updated URL. Now I used it and I got the `refresh_token` as expected 🥳 

<img width="246" alt="Screen Shot 8" src="https://user-images.githubusercontent.com/1402241/106214264-10369a00-6194-11eb-84fc-f124705a84c2.png">

Author: So....everything is good?

Reviewer: The second problem seems fine, I just need to wait a little longer to ensure it doesn’t expire.

But it’s still a breaking change, so a few more changes are required as suggested.

	<img width="771" alt="Create credentials" src="https://user-images.githubusercontent.com/1402241/77865679-e89f3a00-722f-11ea-942d-5245091f22b8.png">

> 0. Select **Chrome App** (or **Desktop app** if available), enter `chrome-webstore-upload` and click **Create** 

Reviewer: Is Other gone? I'll have to verify these steps because Chrome App existed already, but I didn't use it before.

```suggestion
0. Select **Chrome App** (**Desktop app** and **Other** also work, if Chrome App is missing), enter `chrome-webstore-upload` and click **Create** 
```

Author: Yeah last time I checked it's gone.

> 	> <img width="187" alt="Configure client type" src="https://user-images.githubusercontent.com/25856620/103254672-d1f16380-49b8-11eb-8cdf-98c2483be403.png">

Reviewer: Thanks for updating the screenshots!

> 0. Save your ✅ `clientId`, this is your 1 of your 2 keys.

Reviewer: ```suggestion
0. Save your ✅ `clientId`, this is 1 of your 2 keys.
```

> 1. Save your ✅ `clientId`, this is your 1 of your 2 keys. The `clientSecret` will be present if you choose **Desktop App** but in practice you won't need to use it.

Reviewer: Not needed, no need to talk about it
```suggestion
0. Save your ✅ `clientId`, this is your 1 of your 2 keys.
```

> 2. Place your `clientId` in this URL and open it:

Reviewer: ```suggestion
0. Place your `clientId` in this URL and open it:
```

	<img width="771" alt="Create credentials" src="https://user-images.githubusercontent.com/1402241/77865679-e89f3a00-722f-11ea-942d-5245091f22b8.png">

> 0. Select **Chrome App** (**Desktop app** and **Other** also work, if Chrome App is missing), enter `chrome-webstore-upload` and click **Create** 

Author: So that means the `Other` type is still there?

Reviewer: I think we had other people reporting they couldn't find one type or the other, so maybe Google shows different possible values to different people. I think it's ok to specify "Other" as an alternative.

Author: Yeah makes sense 👍

}
```

> 1.    Done. Now you should have ✅ `clientId` and ✅ `refreshToken`. You can use these for all your extensions, but don't share them!

Reviewer: 1 is not [over 9000](https://en.wikipedia.org/wiki/It%27s_Over_9000!)
```suggestion
9001. Done. Now you should have ✅ `clientId` and ✅ `refreshToken`. You can use these for all your extensions, but don't share them!
```
